udata/core/dataset/models.py

```diff
@@ -148,6 +149,48 @@ def check_availability(self, group):
     def is_available(self):
         return self.check_availability(group=None)
 
+    @cached_property
+    def json_ld(self):
+
+        result = {
+            '@type': 'DataDownload',
+            '@id': str(self.id),
+            'url': self.url,
+            'name': self.title or _('Nameless resource'),
+            'contentUrl': self.url,
+            'dateCreated': self.created_at.isoformat(),
+            'dateModified': self.modified.isoformat(),
+            'datePublished': self.published.isoformat(),
+        }
+
+        if 'views' in self.metrics.keys():
```

Comment:
I know, but I think it is a bit more self-understanding like this.
When you read it, you immediately see that metrics is a dictionary. Otherwise, you do not know whether it is a list, a dictionary or any other kind of iterable.
I am ready to correct it if you confirm that this is the current coding guideline of the project.
Let me know !